### PR TITLE
[SP] Add augmented reality sample with Polymer UI

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
BUG=XWALK-6362

### CODE REVIEW

**Comment by halton** [comment]
@huningxin what is this Appveyor error?

**Comment by DonnaWuDongxia** [comment]
Went green! @leonhsl @halton please help to review. Thanks!

**Comment by halton** [comment]
@leonhsl could you spend some time to review?

File: sample/src_polymer/elements/scene-perception-pages/augmented-reality-page/augmented-reality-page.js [hunk 1]
```diff
@@ -0,0 +1,836 @@
...
+        type: Number,
+        value: -1
+      }
+    },
+    _getTrackingState: function(spState) {
+      return spState == 2 ? false : true;
+    },
+    _activatedChanged: function(newValue, oldValue) {
+      if (newValue) {
+        console.log('sp page activated');
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/augmented-reality-page/augmented-reality-page.js]: `reconstruction-page` is also logging `sp page activated` ...

File: sample/src_polymer/elements/scene-perception-pages/augmented-reality-page/augmented-reality-page.js [hunk 2]
```diff
@@ -0,0 +1,836 @@
...
+  const MIN_ACCEPTABLE_QUALITY = 0.25;
+  const SP_SIZE_WIDTH = 320;
+  const SP_SIZE_HEIGHT = 240;
+  const SP_SIZE_FPS = 60;
+  const BOTTOM_PANEL_HEIGHT = 50;
+  var myStatus = new RSUtils.Status();
+  var sp = RSUtils.getSP();
+  if (!sp) return;
+
+  function destroySP(spDom) {
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/augmented-reality-page/augmented-reality-page.js]: Maybe here we can rename `spDom` to `arDom`?

**Comment by leonhsl** [comment]
Other than above comments, lgtm

**Comment by DonnaWuDongxia** [comment]
@leonhsl thanks for reviewing. Updated.

**Comment by leonhsl** [comment]
lgtm, merging..

**Comment by leonhsl** [comment]
Merged

**Pull request merged**